### ISSUE TITLE
multi study query not working when `global_case_list: true`

### ISSUE THREAD

**Issue by inodb** [cBioPortal/cbioportal#4772]
Related to:
https://github.com/cBioPortal/cbioportal/issues/4502

The case list type shows up as `other` when using `global_case_list: true` which cause the multi study query the malfunction.

As reported by:

https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/cbioportal/qXXUX-a7p5I/FmJZeoFRAQAJ

**Comment by fedde-s** [comment]
Is this related to the decision discussed in https://github.com/cBioPortal/cbioportal/issues/4502?

**Comment by fedde-s** [comment]
@kalletlak, is this the query selection logic you mentioned?

**Comment by kalletlak** [comment]
@fedde-s yes. There is a pr to fix it https://github.com/cBioPortal/cbioportal-frontend/pull/1449
